_playwright-tests/UI/RepositoryPackages.spec.ts

```diff
@@ -0,0 +1,95 @@
+import { test, expect } from '@playwright/test';
+import { navigateToRepositories } from './helpers/navHelpers';
+import { deleteAllRepos } from './helpers/deleteRepositories';
+import { closePopupsIfExist, getRowByNameOrUrl } from './helpers/helpers';
+export const randomName = () => (Math.random() + 1).toString(36).substring(2, 6);
+export const repoName = `${randomName()}`;
+export const edited_repo = `${repoName}-Edited`;
```

Comment:
none of these should be exported as you don't want them in other tests, you can just remove the export keyword 🗑️ 

and also, the random name is a bit overkill 😅 and using only that as the name complicates cleanup for you, just a prefix like `snapshot-package-list-test` and then appending edited to it should be fine, then you can just cleanup repos with that prefix 💭😄

Reply:
Thanks ! Fixed